packages/atomic/src/components/common/facets/facet-search/facet-search-input-guard.ts

```diff
@@ -0,0 +1,38 @@
+import {html, nothing} from 'lit';
+import {directive, Directive} from 'lit/directive.js';
+
+const MIN_VALUES_WHERE_FACET_SEARCH_IMPROVES_UX = 9;
+
+interface FacetSearchInputGuardProps {
+  withSearch: boolean;
+  canShowMoreValues: boolean;
+  numberOfDisplayedValues: number;
+}
+
+class FacetSearchInputGuardDirective extends Directive {
```

Comment:
I'm wondering if we should just uses `when` and exposes a "selector" function.

```ts
const shouldDisplayFacetSearchInput = ({withSearch,canShowMoreValues,numberOfDisplayedValues}: FacetSearchInputGuard) =>
  wtihSearch && !(
    !canShowMoreValues &&
    numberOfDisplayedValues < MIN_VALUES_WHERE_FACET_SEARCH_IMPROVES_UX
  );
  
 when(shouldDisplayFacetSearchInput(props), content);
 ```

Reply:
TBH, I really like that!